### DISCUSSION TITLE
When does a new release is distributed on F-Droid ?

### DISCUSSION THREAD

**Discussion by ydrolet** [koreader/koreader#8865] [Q&A] [answered]
Hi! How long habitually does it take for a new release to be distributed on F-Droid ? [2022.02](https://github.com/koreader/koreader/releases/tag/v2022.02) was released on GitHub last Sunday but the last version on [F-Droid](https://f-droid.org/en/packages/org.koreader.launcher.fdroid/) is still 2022.01.

**Comment by Frenzie** [comment, marked as answer]
It's an automated process. It was picked up fairly quickly but I don't know how the build queuing works exactly. It's probably explained somewhere on F-Droid. Anyway, past experience suggests it can be as fast as a day or two or it can take some 5-7 days.

https://gitlab.com/fdroid/fdroiddata/-/blob/51425f09cfd622f35dd6c41f22198fc25a99bd9b/metadata/org.koreader.launcher.fdroid.yml
